### PR TITLE
Update spanish translation 2023-09-16

### PR DESCRIPTION
**Pull request by SlimRunner**
A smaller update to the Spanish translations. I also added a little nitpick in one of the source English tokens I translated from.

### CODE REVIEW

File: localization/en.ftl [hunk 1]
```diff
@@ -243,7 +243,7 @@ manage-metadata-desc = Manage Metadata, such as GLesmos or pinned/unpinned statu
 intellisense-name = Intellisense
 intellisense-desc = Brings several common IDE features to Desmos, including autocompletion suggestions, function call help, and jump to definition. Documentation here:
 intellisense-opt-subscriptify-name = Auto-Subscriptify
-intellisense-opt-subscriptify-desc = Automatically add subscripts to variable/function names when they are typed without subscripts.
+intellisense-opt-subscriptify-desc = Automatically predicts text of variables/functions with subscripts when they are typed without subscripts.
```
Comment (jared-hughes) [inline localization/en.ftl:246]: "predicts" sounds like it's doing more than it does.
Maybe "converts names to subscripts"? All it's does is convert `fxyz` to `f_{xyz}` sometimes.

**Review by jared-hughes — CHANGES_REQUESTED** [review]
Looks good! Though not sure about the "predicts" wording. Agree the current phrasing is a bit odd, since "add subscripts" sounds like it's inserting new letters.

File: localization/en.ftl (hunk 1 above)
Comment (SlimRunner) [reply localization/en.ftl]: I changed the word to "converts". I also updated the Spanish version accordingly.

**Comment by SlimRunner** [comment]
I noticed there are several incoming PRs with changes to en.ftl. It could be better to hold of on this one until a few more of them are merged (or expected to be merged).

If that is the case, I can proactively amend changes to this same PR for the expected changes.

**Comment by jared-hughes** [comment]
The less-f-spacing PR is in. Good time to update this PR.

**Comment by jared-hughes** [comment]
Also the emit config for text mode https://github.com/DesModder/DesModder/pull/706/files#diff-001b525aa605feef6a113ce3152bbf39a5bb34a67f44fd29157f2654f4b6aafc

**Review by jared-hughes — APPROVED** [review]

**Pull request merged**